test/functional/cancel.test.ts

```diff
@@ -20,14 +20,6 @@ const initialize = async () => {
 beforeEach(() => initialize());
 afterEach(() => polkadotApi.disconnect());
 
-test('Cancel failed with incorrect format taskID', async () => {
```

Comment:
The **incorrect format** taskId should still be tested during cancellation. Is it covered by other tests already?

Reply:
Right, with the new taskId "a string here" won’t be able to find anything.